tests/lax_numpy_test.py

```diff
@@ -323,6 +323,8 @@ def testCountNonzero(self, shape, dtype, axis):
 
   @jtu.sample_product(shape=all_shapes, dtype=all_dtypes)
   def testNonzero(self, shape, dtype):
+    if numpy_version >= (2, 0, 0) and not shape:
+      shape = (1,)  # NumPy removed support for nonzero on scalars in 2.0.
```

Comment:
A better fix would be to replace `all_shapes` above with `nonzerodim_shapes`.

This has been deprecated since december, and I was planning on finalizing the deprecation sometime this week, so we may as well future-proof the test.